### PR TITLE
Fixed escaping in class-wc-admin-dashboard.php

### PR DESCRIPTION
**Pull request by IamYashKapoor**
### All Submissions:

* [x] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/master/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:
# Fixed escaping in class-wc-admin-dashboard.php

### How to test the changes in this Pull Request:

1. If the value is "><script>alert();</script> and you tried to output it in an HTML attribute it would close the current HTML tag and open a script tag. This is unsafe. By escaping the value it won't be able to close the HTML attribute and tag and output unsafe HTML.
2. Now it will be fixed.

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [x] Have you written new tests for your changes, as applicable?
* [x] Have you successfully run tests with your changes locally?

### Changelog entry

> Fixed escaping in class-wc-admin-dashboard.php

### CODE REVIEW

File: includes/admin/class-wc-admin-dashboard.php [hunk 1]
```diff
@@ -345,7 +345,7 @@ public function network_orders() {
 			<div class="post-type-shop_order">
 			<div id="woocommerce-network-order-table-loading" class="woocommerce-network-order-table-loading is-active">
 				<p>
-					<span class="spinner is-active"></span> <?php esc_html_e( 'Loading network orders', 'woocommerce' ); ?>
+					<span class="spinner is-active"></span><?php esc_html_e( 'Loading network orders', 'woocommerce' ); ?>
```
Comment (rodrigoprimo) [inline includes/admin/class-wc-admin-dashboard.php]: Could you please clarify why did you opt to remove this space? I didn't test to see if it has any consequences (and I assume it is unlikely to have) but asking anyway as this change doesn't seem related to this PR and it is not mentioned in the PR description, so maybe it was added by mistake?

Comment (yash-webkul) [reply includes/admin/class-wc-admin-dashboard.php]: Yes, it was removed by mistake, now I have added it back again.

**Review by rodrigoprimo — APPROVED** [review]
Thanks for the clarification and change. Looks good now.
